StantUiIosLibrary/Classes/Cards/Contacts/ContactsCard.swift

```diff
@@ -35,85 +32,92 @@ public class ContactsCard: UITableViewCell {
         photoImageView = nil
     }
     
-    public func configureViewFor(construction: ContactsInformation) {
+    public func configureViewFor(contacts: ContactsInformation) {
```

Comment:
Change the parameter name to contact. It's a single element, so it should be singular.

Reply:
Done